code/modules/cargo/orderconsole.dm

```diff
@@ -280,6 +284,17 @@
 				if(dept_choice == "Cargo Budget")
 					personal_department = null
 
+		if(!account || !id_card || !living_user || !access)
+			living_user = user
+			id_card = living_user.get_idcard(TRUE)
```

Comment:
what's going on with this?

Reply:
my gut is we should just do the living check/typecast twice and not hoist the vars up like this